testfile.txt

```diff
@@ -0,0 +1,7 @@
+I wonder if the whole class can see this on Junipers branch now.
+
+Adding some extra info here.
+
```

Comment:
Hey can you get rid of this new line here

Reply:
Hey could we take out the extra lines here?

Reply:
Change made